plugins/node_peer_count/main.go

```diff
@@ -26,12 +27,14 @@ var (
 	addr    string
 	port    int
 	minPeer int
+	tenderPort uint
 )
 
 func init() {
 	flag.StringVar(&addr, "addr", defaultAddr, "IP Address(e.g. 0.0.0.0, 127.0.0.1)")
 	flag.IntVar(&port, "port", defaultPort, "Port number, default 9091")
 	flag.IntVar(&minPeer, "minPeer", defaultPeer, "minimum peer count, default 5")
+	flag.UintVar(&tenderPort, "tendermintPort", defaultPeer, "tendermint port, default 26657")
```

Comment:
I think `defaultPeer` have to change another variable.

Reply:
Thanks, I modified.